command/base.go

```diff
@@ -411,6 +418,15 @@ func (c *BaseCommand) flagSet(bit FlagSetBit) *FlagSets {
 			}
 			f.StringVar(agentAddrStringVar)
 
+			proxyAddrStringVar := &StringVar{
+				Name:       "proxy-address",
+				Target:     &c.flagVaultProxyAddress,
+				EnvVar:     api.EnvVaultProxyAddr,
+				Completion: complete.PredictAnything,
+				Usage:      "Address of the Vault Proxy.",
```

Comment:
If we do merge this as-is, this should be updated to something like the following:
`HTTP or HTTPS proxy location which should be used by all requests to access Vault.`